runner/execer/os/os_execer.go

```diff
@@ -14,9 +16,13 @@ func NewExecer() execer.Execer {
 type osExecer struct{}
 
 func (e *osExecer) Exec(command execer.Command) (result execer.Process, err error) {
+	if len(command.Argv) <= 0 {
```

Comment:
I'd just say == 0; if len starts returning negative numbers, everything is awful.

(But that's so far below a nit...)